### PR TITLE
Update to queries

### PR DESCRIPTION
**Pull request by radeonxray**
- Updated queries to correctly retrieve comments.
- Also added the helge_id as a return value, as it is needed in order for the frontend to post comments to the correct post/topics/stories. [UPDATE: providing a helge_id is no longer needed! New methods created and the frontend will always give a 0 atm.]
- Changed /latest to also include both post and comment table to find the latest helge_id to be returned
- Added reject/error-handling to missing queries

### CODE REVIEW

File: src/controllers/mysql/queries/commentsQueries.ts [hunk 1]
```diff
@@ -17,12 +17,30 @@ export function selectAllComments(){
     })
   }
 
+  //Retrives all comments based on the ID
+  export function selectAllCommentsFromPostId(postId: number){
```
Comment (Viterkim) [inline src/controllers/mysql/queries/commentsQueries.ts]: needs reject with error handling

File: src/controllers/mysql/queries/postAndCommentsQueries.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { resolve } from 'url';
 //Retrieves all post and comments by Username
 export function selectAllContentByUsername(username:string){
```
Comment (Viterkim) [inline src/controllers/mysql/queries/postAndCommentsQueries.ts]: needs error handling

File: src/controllers/mysql/queries/postAndCommentsQueries.ts [hunk 3]
```diff
@@ -20,7 +20,7 @@ export function selectAllContentByUsername(username:string){
 //Retrieves all post and comments by User ID
 export function selectAllContentByUserId(userid:number){
```
Comment (Viterkim) [inline src/controllers/mysql/queries/postAndCommentsQueries.ts]: error handling

File: src/controllers/mysql/queries/commentsQueries.ts (hunk 1 above)
Comment (radeonxray) [reply src/controllers/mysql/queries/commentsQueries.ts]: Fixed

File: src/controllers/mysql/queries/postAndCommentsQueries.ts (hunk 3 above)
Comment (radeonxray) [reply src/controllers/mysql/queries/postAndCommentsQueries.ts]: Fixed

File: src/controllers/mysql/queries/postAndCommentsQueries.ts (hunk 2 above)
Comment (radeonxray) [reply src/controllers/mysql/queries/postAndCommentsQueries.ts]: Fixed

**Pull request merged**